src/kOS.Safe/Compilation/Opcode.cs

```diff
@@ -1875,11 +1875,12 @@ public OpcodePushDelegateRelocateLater(string destLabel, bool withClosure) : bas
         protected OpcodePushDelegateRelocateLater() : base() {}
         
         public override void PopulateFromMLFields(List<object> fields)
-        {
+        {            
             // Expect fields in the same order as the [MLField] properties of this class:
             if (fields == null || fields.Count<1)
                 throw new Exception("Saved field in ML file for OpcodePushDelegateRelocatelater seems to be missing.  Version mismatch?");
-            WithClosure = (bool)fields[0];
+            DestinationLabel = (string)( fields[0] ); // this is really from the base class.
+            WithClosure = (bool)fields[1];
```

Comment:
do you expect to get a lot of use out of derived classes in opcodes? if not i think we should seal/final the classes to prevent confusion.